Code/Framework/AzQtComponents/AzQtComponents/Application/AzQtApplication.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * All or portions of this file Copyright (c) Amazon.com, Inc. or its affiliates or
+ * its licensors.
+ *
+ * For complete copyright and license terms please see the LICENSE at the root of this
+ * distribution (the "License"). All use of this software is governed by the License,
+ * or, if provided, by the license below or the license accompanying this file. Do not
+ * remove or modify any license notices. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ */
+
+#include <AzQtComponents/Application/AzQtApplication.h>
+
+#include <AzCore/PlatformIncl.h> // This should be the first include to make sure Windows.h is defined with NOMINMAX
+#include <AzQtComponents/Utilities/QtPluginPaths.h>
+
+namespace AzQtComponents
+{
+
+    AzQtApplication::AzQtApplication(int& argc, char** argv)
+        : QApplication(argc, argv)
+    {
+         // Use a common Qt settings path for applications that don't register their own application name
+         QApplication::setOrganizationName("O3DE");
```

Comment:
Awesome to see this!

There is also the EditorQtApplication class, which is the base QApplication used for the Editor. There are a lot of duplicated things done in that class that I think could be done in this base class as well (e.g. it also sets the same organization/application name). There is likely some amount of code in EditorQtApplication that can just be removed, and whatever is left that doesn't make sense to be moved to the base class can stay in EditorQtApplication, but it could then be changed to inherit from your new AzQtApplication instead of QApplication.